auth/http.go

```diff
@@ -598,10 +602,12 @@ func getUserObjectsJSON(central *ImqsCentral, users []authaus.AuthUser, ident2pe
 		} else if err != nil {
 			central.Central.Log.Warnf("issue fetching group names for user %v : %v", user.UserId, err)
 		}
-		userStats, err := central.Central.GetUserStats(user.UserId)
+
 		if err != nil {
 			return nil, err
 		}
+		userStat, _ := userStats[user.UserId]
+
```

Comment:
```suggestion
		userStat, ok := userStats[user.UserId]
		if !ok {
		// Maybe some warning logging that the user is not valid
		// We will then need to continue in this for-loop.
		}

```

I don't know if it should be a hard fail or just a warning if we can't find a user in userStats?

Reply:
There will be a lot of users with no entries in UserStats - for not having logged in, or not having their permissions changed at all. So if we put something in here then we'll spam the logs. @anzelIMQS

Reply:
From what I see userStat cannot be nil - because it is not a pointer. And subsequent accesses to its fields produce zero-values and zero-equivalent strings:

```
func TestUserStats(t *testing.T) {
	userStats := make(map[UserId]UserStats, 100)
	stat, ok := userStats[123]
	assert.False(t, ok)
	assert.NotNil(t, stat)
	assert.NotNil(t, stat.DisabledDate)
	assert.False(t, stat.DisabledDate.Valid)
	assert.NotNil(t, stat.DisabledDate)
	assert.NotEqual(t, "", fmt.Sprintf("%s", stat.DisabledDate.Time))
	fmt.Printf("%+v \n", stat)
}
```